docker/Dockerfile_dev

```diff
@@ -19,12 +19,12 @@ FROM ghcr.io/scientificcomputing/fenics:2023-01-16
 #RUN export FENICS_SRC_DIR=/usr/local/src && \
 #    cd /usr/local/src && \
 #    export GIT_LFS_SKIP_SMUDGE=1 && \
-#    git clone https://github.com/justinlaughlin/dolfinstubs.git && \
-#    cd dolfinstubs && \
+#    git clone https://github.com/justinlaughlin/dolfinsmart.git && \
```

Comment:
```suggestion
#    git clone https://github.com/RangamaniLabUCSD/smart && \
```
or similar in due time

Reply:
Nope! I agree, let's clean up the items commented out